packages/beacon-node/src/metrics/validatorMonitor.ts

```diff
@@ -40,6 +40,7 @@ export type ValidatorMonitor = {
   registerLocalValidatorInSyncCommittee(index: number, untilEpoch: Epoch): void;
   registerValidatorStatuses(currentEpoch: Epoch, statuses: AttesterStatus[], balances?: number[]): void;
   registerBeaconBlock(src: OpSource, seenTimestampSec: Seconds, block: allForks.BeaconBlock): void;
+  registerBlobSideCar(src: OpSource, seenTimestampSec: Seconds, blob: deneb.BlobSidecar): void;
```

Comment:
```suggestion
  registerBlobSidecar(src: OpSource, seenTimestampSec: Seconds, blob: deneb.BlobSidecar): void;
```

I think "Sidecar" is treated as a single word

Reply:
yes, thanks for pointing it out, updated :slightly_smiling_face: